src/test/java/org/folio/linked/data/integration/kafka/sender/search/WorkUpdateMessageSenderTest.java

```diff
@@ -107,8 +119,36 @@ void sendWorkUpdated_shouldSendDelete_ifNewWorkIsNotIndexableButOldIs() {
     producer.produce(oldResource, newResource);
 
     // then
-    verify(deleteEventProducer)
-      .accept(oldResource);
-    verifyNoInteractions(resourceIndexEventMessageProducer, createEventProducer);
+    verify(eventPublisher).publishEvent(new ResourceDeletedEvent(oldResource));
+    verifyNoInteractions(resourceIndexEventMessageProducer);
```

Comment:
looks like this line contradicts with test case name

Reply:
correct, fixed, thx!